README.md

```diff
@@ -41,217 +41,1885 @@ The required view creation scripts are found in:
 
 ```sql
 SELECT
-    id, machine_name, name, value, description, created_at
+    id, automation_system_name, config_key, config_value, description, created_at
 FROM
-    `biosero_test`.`configurations`
+    `biosero_uat`.`configurations`
 WHERE
-    machine_name = 'Workcell 1'
+    machine_name = 'CPA'
 ORDER BY id ASC;
 ```
 
-Example JSON representation for workcell 1 configuration:
+Example JSON representation for CPA configuration:
```

Comment:
What is this referring to? The different config_key 's ?

Reply:
The set of config keys for workcell 'CPA', will make clearer